src/plugins/intel_cpu/src/utils/shape_inference/shape_inference.cpp

```diff
@@ -563,6 +549,8 @@ std::shared_ptr<IShapeInfer> make_shape_inference(const std::shared_ptr<ngraph::
         return std::make_shared<entryFallbackWithPadding<ov::op::v1::DeformableConvolution>>(node);
     } else if (auto node = ov::as_type_ptr<ov::op::v8::DeformableConvolution>(op)) {
         return std::make_shared<entryFallbackWithPadding<ov::op::v8::DeformableConvolution>>(node);
+    } else if (auto node = ov::as_type_ptr<ov::opset8::Transpose>(op)) {
+        return make_shared_entryIO(node);
```

Comment:
must accept [constant input data](https://github.com/openvinotoolkit/openvino/pull/12937/files#diff-5cb53918d9dcf47e61e0920169f8ac943845707f77023296ab6d997818c104d6R128)
```suggestion
        return make_shared_entryIOC(node);
```

Reply:
Done.